client/lib/inflight/index.js

```diff
@@ -1,16 +1,24 @@
-var inflight = {};
+const inflight = new Set();
 
 // Utility methods to track inflight async requests
-module.exports = {
+export default {
```

Comment:
@blowery: what about using named exports _and_ exporting an object with all the funcs.  that way its backwards compatible and we don't need to go mucking around with all the clients